website/_docs/index.cn.md

```diff
@@ -9,21 +9,31 @@ permalink: /cn/docs/index.html
 ------------  
 > Extreme OLAP Engine for Big Data
 
-Apache Kylin™是一个开源的分布式分析引擎，提供Hadoop之上的SQL查询接口及多维分析（OLAP）能力以支持超大规模数据，最初由eBay Inc.开发并贡献至开源社区。
+Apache Kylin™ 是一个开源 OLAP（On-Line Analytics Processing）分析引擎，针对数据库与亿万级记录提供亚秒级查询响应能力。2014 年由 eBay Inc.开发并贡献至开源社区。2015 年毕业于 Apache 顶级项目并迅速被全球数千家公司与组织应用于数据分析业务，同时也在 2015 与 2016 年赢得 InfoWorld Bossie Awards 年度最佳大数据开源工具。 
```

Comment:
"Online" is a word, not "On-Line", please check the terms: https://en.wikipedia.org/wiki/Online_analytical_processing